client/src/partials/reports/global_transaction/global_transaction.js

```diff
@@ -57,7 +58,9 @@ angular.module('bhima.controllers')
         account.account_number = String(account.account_number);
       });
       $scope.model.c = $scope.enterprise.currency_id;
```

Comment:
Why not change `$scope.model.c` to `$scope.currencyId`?  They are equal length (to type), and $scope.currencyId is one less level deep in the $scope object.  Also, $scope.currencyId is much more readable.